prow/cluster.yaml

```diff
@@ -823,3 +823,37 @@ spec:
   - port: 80
     targetPort: 8888
   type: NodePort
+---
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: default
+  name: "tide"
```

Comment:
You'll also need to add `serviceAccountName: tide` to the Tide deployment.

Reply:
good catch, done